R/kendrickMass.R

```diff
@@ -0,0 +1,106 @@
+#' @title Calculate Kendrick masses from exact masses
+#'
+#' @description
+#'
+#' `calculateKendrickMass` Calculates the Kendrick mass from exact masses.
+#'
+#' @param x `numeric` exact masses
+#'
+#' @return `numeric` calculated Kendrick masses
+#'
+#' @author Michael Witting
+#'
+#' @export
+#'
+#' @examples
+#'
+#' calculateKendrickMass(760.5851)
+#'
+calculateKendrickMass <- function(x) {
+  
+  x * (14.00000 / 14.01565)
+  
+}
+
+#' @title Calculate Kendrick mass defects from exact masses
+#'
+#' @description
+#'
+#' `calculateKendrickMassDefect` Calculates the Kendrick mass defects from exact masses.
+#'
+#' @param x `numeric` exact masses
+#'
+#' @return `numeric` calculated kendrick mass defects
+#'
+#' @author Michael Witting
+#'
+#' @export
+#'
+#' @examples
+#'
+#' calculateKendrickMassDefect(760.5851)
+#'
+calculateKendrickMassDefect <- function(x) {
+  
+  kendrick <- calculateKendrickMass(x)
+  nominalKendrick <- as.integer(kendrick)
```

Comment:
is `as.integer` here correct? below you use `round` instead.

Reply:
Yes, `as.integer` is correct, because we need the nominal mass